packages/web-twig/src/Factory/TwigFactory.php

```diff
@@ -30,6 +31,7 @@ public function create(): Environment
         $this->twig->setLoader($this->loader);
 
         $this->twig->addGlobal('_spiritClassPrefix', $this->classPrefix);
+        $this->twig->addGlobal('_spiritCssVariablePrefix', $this->cssVariablePrefix);
```

Comment:
🤔 maybe it would be more correct to format it like this 🤔 

```suggestion
        $this->twig->addGlobal('_spiritCSSVariablePrefix', $this->cssVariablePrefix);
```

Reply:
renamed